City-of-Helsinki/helsinki-paatokset
# PP-0: Drupal updates for week 38

**To test:**
- Checkout branch
- Run `make fresh` (if you have an existing database dump)
- Run `make drush-updb` to make sure there aren't any pending database updates
- Make sure all modules and Drupal core is actually up-to-date: https://helsinki-paatokset.docker.so/fi/admin/reports/updates
- Test that the site works and nothing weird was introduced in the platform config updates.

**Updates:**
```
  - Upgrading drupal/admin_toolbar (2.4.0 => 2.5.0)
  - Upgrading drupal/core (9.2.4 => 9.2.6)
  - Upgrading drupal/core-composer-scaffold (9.2.4 => 9.2.6)
  - Upgrading drupal/core-dev (9.2.4 => 9.2.6)
  - Upgrading drupal/core-recommended (9.2.4 => 9.2.6)
  - Upgrading drupal/entity_usage (2.0.0-beta4 => 2.0.0-beta5)
  - Upgrading drupal/hdbt (1.3.11 => 1.3.20)
  - Upgrading drupal/hdbt_admin (1.3.7 => 1.3.9)
  - Upgrading drupal/helfi_api_base (1.1.9 => 1.2.0)
  - Upgrading drupal/helfi_platform_config (1.3.1 => 1.3.8)
  - Upgrading drupal/helfi_tpr (1.3.4 => 1.3.7)
  - Upgrading drupal/token_filter (1.2.0 => 1.3.0)
```
